alpine-rails-nginx/Dockerfile

```diff
@@ -15,44 +13,28 @@ RUN echo '@testing http://dl-4.alpinelinux.org/alpine/edge/testing' >> /etc/apk/
   | xargs apk add --update-cache \
   && rm -- /tmp/apk-packages
 
-# Install nginx
-COPY install-nginx.sh /tmp/
-COPY nginx-remove-server-name.patch /tmp/
-RUN /tmp/install-nginx.sh \
-  && rm -rf -- /usr/local/src /tmp/*
-
-# Upgrade bundler
-RUN echo 'gem: --no-document' > ~/.gemrc \
-  && gem install bundler
-
-# Add utility scripts
-COPY setup-directories.sh /root/
-
-# Setup system configuration
-COPY etc /etc/
+# Create system users
+RUN adduser -DSH -G nogroup nginx \
+  && adduser -DSH -G nogroup rails
```

Comment:
Should we change this to a dedicated `rails` / `nginx` group instead the shared `nogroup` to avoid accidental set group permissions to cause read/writable files to unintend users?